### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by halkeye**
* Pin the version of wine to 5.11 which doesn't have the wine tricks issue
* force steamcmd to download the windows version

### CODE REVIEW

**Comment by halkeye** [comment]
No matter what combo of ip and cfg and stuff, I can't get it to startup. So maybe someone who had experience in the past can provide insight

```
space_1  | + exec wine64 SpaceEngineersDedicated.exe -noconsole -ignorelastsession -ip 172.16.10.2 -path 'C:\users\root\AppData\Roaming\SpaceEngineersDedicated'
space_1  | 00c8:err:vulkan:wine_vk_init Failed to load Wine graphics driver supporting Vulkan.
space_1  | 00c8:err:module:find_forwarded_export module not found for forward 'winevulkan.wine_vkGetInstanceProcAddr' used by L"C:\\windows\\system32\\vulkan-1.dll"
space_1  | 2020-09-30 06:25:36.376: App Version: 01_196_016
space_1  | 2020-09-30 06:25:36.382: Is official: True [NO][NIS][NAMP]
space_1  | 2020-09-30 06:25:36.382: Environment.ProcessorCount: 8
space_1  | 2020-09-30 06:25:36.390: Environment.OSVersion: Microsoft Windows 7 Professional (Microsoft Windows NT 6.1.7601 Service Pack 1)
space_1  | 2020-09-30 06:25:36.390: Environment.CommandLine: "C:\SpaceEngineersDedicatedServer\DedicatedServer64\SpaceEngineersDedicated.exe" -noconsole -ignorelastsession -ip 172.16.10.2 -path C:\users\root\AppData\Roaming\SpaceEngineersDedicated
space_1  | 2020-09-30 06:25:36.390: Environment.Is64BitProcess: True
space_1  | 2020-09-30 06:25:36.390: Environment.Is64BitOperatingSystem: True
space_1  | 2020-09-30 06:25:36.391: Environment.Version: .NET Framework 4.7.3062.0
space_1  | 2020-09-30 06:25:36.391: Environment.CurrentDirectory: C:\SpaceEngineersDedicatedServer\DedicatedServer64
space_1  | 2020-09-30 06:25:36.391: CPU Info: Intel(R) Xeon(R) CPU           L5410  @ 2.33GHz
space_1  | Computer does not meet minimum requirements, forcing Experimental mode
space_1  | 00c8:err:combase:RoGetActivationFactory Failed to find library for L"Windows.Foundation.Diagnostics.AsyncCausalityTracer"
space_1  | 2020-09-30 06:25:40.518: Bind IP : 172.16.10.2:27016
space_1  | 00c8:err:winsock:interface_bind Failed to bind to interface, receiving broadcast packets will not work on socket 1614.
space_1  | dbus[72]: arguments to dbus_connection_send_with_reply_and_block() were incorrect, assertion "connection != NULL" failed in file ../../../dbus/dbus-connection.c line 3543.
space_1  | This is normally a bug in some application using the D-Bus library.
space_1  |
space_1  |   D-Bus not built with -rdynamic so unable to print a backtrace
space_1  | wine: Assertion failed at address 00007F6AE7781F47 (thread 0064), starting debugger...
space_1  | 0158:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
space_1  | 0158:err:winediag:nodrv_CreateWindow Make sure that your X server is running and that $DISPLAY is set correctly.
space_1  | 2020-09-30 06:25:56.340: Error: No IP assigned.
space_1  | 2020-09-30 06:25:56.404: Exception while reporting crash.
space_1  | 2020-09-30 06:25:56.404: System.InvalidCastException: Unable to cast object of type 'System.Boolean' to type 'System.String'.
space_1  |    at Sandbox.MyInitializer.OnCrash(String logPath, String gameName, String minimumRequirementsPage, Boolean requiresDX11, Exception exception, IntPtr exceptionPointers, Boolean oom)
```

**Comment by Lazalatin** [comment]
I will try that one now. Maybe I can provide some insight afterwards 👍🏻

File: entrypoint.sh [hunk 1]
```diff
@@ -37,6 +37,7 @@ fi
 
 steamcmd \
   +login anonymous \
+  +@sSteamCmdForcePlatformType windows \
```
Comment (Lazalatin) [inline entrypoint.sh:40]: So, it seems correct, that this option enables building the image under Windows.
But, as this repository tries to achieve that SpaceEngineers Server runs under a linux kernel based environment, I would suggest to revert this PR to commit 8fdeb87e20cb5565bda07e903c330fb2e0b2d20a and entirely remove this line.

**Comment by Lazalatin** [comment]
Besides my comment above this PR works greatly 👍🏻 Thank you for your contribution ❤️

File: entrypoint.sh (hunk 1 above)
Comment (halkeye) [reply entrypoint.sh:40]: If you remove that line, then it downloads a linux binary, which doesn't work in wine
at least for me

Comment (halkeye) [reply entrypoint.sh:40]: huh, maybe not... i don't know why i added it then

Comment (halkeye) [reply entrypoint.sh:40]: git revert 55cc46732ecdbd589c7124ef36e2dee88d4be687

docker-compose up --build
space_1  | Connecting anonymously to Steam Public...Logged in OK
space_1  | Waiting for user info...OK
space_1  | ERROR! Failed to install app '298740' (Invalid platform)

Comment (halkeye) [reply entrypoint.sh:40]: yea, with this force, it'll download the version. Maybe you were testing in an environment where you already had it download?

Comment (halkeye) [reply entrypoint.sh:40]: New Directory:
```
docker-compose up --build
```

```
space_1  |  Update state (0x61) downloading, progress: 99.52 (3427660982 / 3444244797)
space_1  |  Update state (0x101) committing, progress: 30.73 (1058382446 / 3444244797)
space_1  |  Update state (0x101) committing, progress: 66.39 (2286564080 / 3444244797)
space_1  |  Update state (0x101) committing, progress: 87.89 (3027195028 / 3444244797)
space_1  | Success! App '298740' fully installed.
space_1  | 0050:err:explorer:initialize_display_settings Failed to query current display settings for L"\\\\.\\DISPLAY1".
space_1  | 00c4:err:vulkan:wine_vk_init Failed to load Wine graphics driver supporting Vulkan.
space_1  | 00c4:err:module:find_forwarded_export module not found for forward 'winevulkan.wine_vkGetInstanceProcAddr' used by L"C:\\windows\\system32\\vulkan-1.dll"
space_1  | 2020-10-05 20:06:17.786: App Version: 01_196_016
space_1  | 2020-10-05 20:06:17.792: Is official: True [NO][NIS][NAMP]
space_1  | 2020-10-05 20:06:17.792: Environment.ProcessorCount: 8
space_1  | 2020-10-05 20:06:17.800: Environment.OSVersion: Microsoft Windows 7 Professional (Microsoft Windows NT 6.1.7601 Service Pack 1)
space_1  | 2020-10-05 20:06:17.800: Environment.CommandLine: "C:\SpaceEngineersDedicatedServer\DedicatedServer64\SpaceEngineersDedicated.exe" -noconsole -ignorelastsession -ip 0.0.0.0 -path C:\users\root\AppData\Roaming\SpaceEngineersDedicated
space_1  | 2020-10-05 20:06:17.800: Environment.Is64BitProcess: True
space_1  | 2020-10-05 20:06:17.800: Environment.Is64BitOperatingSystem: True
space_1  | 2020-10-05 20:06:17.801: Environment.Version: .NET Framework 4.7.3062.0
space_1  | 2020-10-05 20:06:17.801: Environment.CurrentDirectory: C:\SpaceEngineersDedicatedServer\DedicatedServer64
space_1  | 2020-10-05 20:06:17.801: CPU Info: Intel(R) Xeon(R) CPU           L5410  @ 2.33GHz
space_1  | Computer does not meet minimum requirements, forcing Experimental mode
space_1  | 00c4:err:combase:RoGetActivationFactory Failed to find library for L"Windows.Foundation.Diagnostics.AsyncCausalityTracer"
space_1  | 2020-10-05 20:06:22.008: Bind IP : 0.0.0.0:27016
space_1  | 0148:err:winsock:interface_bind_check No cache entry for ifindex 0.
space_1  | dbus[92]: arguments to dbus_connection_send_with_reply_and_block() were incorrect, assertion "connection != NULL" failed in file ../../../dbus/dbus-connection.c line 3543.
space_1  | This is normally a bug in some application using the D-Bus library.
space_1  |
space_1  |   D-Bus not built with -rdynamic so unable to print a backtrace
space_1  | wine: Assertion failed at address 00007FA55CBDBF47 (thread 0064), starting debugger...
space_1  | 0154:err:winediag:nodrv_CreateWindow Application tried to create a window, but no driver could be loaded.
space_1  | 0154:err:winediag:nodrv_CreateWindow Make sure that your X server is running and that $DISPLAY is set correctly.
space_1  | 2020-10-05 20:06:38.224: Error: No IP assigned.
space_1  | 2020-10-05 20:06:38.283: Exception while reporting crash.
space_1  | 2020-10-05 20:06:38.283: System.InvalidCastException: Unable to cast object of type 'System.Boolean' to type 'System.String'.
space_1  |    at Sandbox.MyInitializer.OnCrash(String logPath, String gameName, String minimumRequirementsPage, Boolean requiresDX11, Exception exception, IntPtr exceptionPointers, Boolean oom)
```
```
$ docker-compose version; docker version; uname -a

docker-compose version 1.22.0, build f46880fe
docker-py version: 3.4.1
CPython version: 3.6.6
OpenSSL version: OpenSSL 1.1.0f  25 May 2017
Client: Docker Engine - Community
 Version:           19.03.13
 API version:       1.40
 Go version:        go1.13.15
 Git commit:        4484c46d9d
 Built:             Wed Sep 16 17:03:45 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.13
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.15
  Git commit:       4484c46d9d
  Built:            Wed Sep 16 17:02:21 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.3.7
  GitCommit:        8fba4e9a7d01810a393d5d25a3621dc101981175
 runc:
  Version:          1.0.0-rc10
  GitCommit:        dc9208a3303feef5b3839f4323d9beb36df0a9dd
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683
Linux odin 3.10.0-1127.19.1.el7.x86_64 #1 SMP Tue Aug 25 17:23:54 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
```

Comment (Lazalatin) [reply entrypoint.sh:40]: Hi, thank you for testing this further!
This looks good so far. Maybe we should take the "No IP assigned" error into another place. It seems to be related to #14 
I will merge this now, so the build works again 👍🏻

Comment (halkeye) [reply entrypoint.sh:40]: sounds good. I ended up with a libvirt image running windows 2019 server for now

Comment (Lazalatin) [reply entrypoint.sh:40]: I can totally understand that. As of now I think this is the best solution.

**Comment by Lazalatin** [comment]
As I only wanted to use the first commit, I manually cherry-picked it into the current default branch.
This PR will be closed but can be considered merged! ;)

**Pull request closed without merging**